stdlib/zipfile.pyi

```diff
@@ -24,17 +35,57 @@ class ZipExtFile(io.BufferedIOBase):
     mode: str
     name: str
     if sys.version_info >= (3, 7):
+        @overload
+        def __init__(
+            self, fileobj: _ClosableZipStream, mode: str, zipinfo: ZipInfo, pwd: Optional[bytes], close_fileobj: Literal[True]
+        ) -> None: ...
+        @overload
         def __init__(
-            self, fileobj: IO[bytes], mode: str, zipinfo: ZipInfo, pwd: Optional[bytes] = ..., close_fileobj: bool = ...
+            self,
+            fileobj: _ClosableZipStream,
+            mode: str,
+            zipinfo: ZipInfo,
+            pwd: Optional[bytes] = ...,
+            *,
+            close_fileobj: Literal[True],
+        ) -> None: ...
```

Comment:
Why is this overload needed? The first overload accepts keyword arguments too.

Reply:
Since `pwd` is (lower-case) optional. Without this overload, you can't leave `pwd` out.